src/Symfony/Component/Security/Http/Firewall/AccessListener.php

```diff
@@ -59,6 +58,15 @@ public function handle(GetResponseEvent $event)
             return;
         }
 
+        if (null !== $this->authorizationChecker) {
+            $this->authorizationChecker->isGranted($attributes, $request);
```

Comment:
shouldn't this be "if (!...isGranted()) throw..."?

Reply:
True, I've adjusted the code